packit_service/log_versions.py

```diff
@@ -0,0 +1,46 @@
+import logging
+
+# from packit import __version__ as packit_version
+
+from ogr import __version__ as ogr_version
+from packit_service import __version__ as ps_version
+from sqlalchemy import __version__ as sqlal_version
+from flask_restx import __version__ as restx_version
+from flask import __version__ as flask_version  # type: ignore
+
+
+# For some reason mypy has an issue with flask's __version__ even though it works completely fine.
+# The "type: ignore" comment causes mypy to ignore that line.
+
+
+logger = logging.getLogger(__name__)
+
+
+# does the actual logging
```

Comment:
These comments above the function signatures seem to be better suited as doc-strings.